bleak/backends/bluezdbus/utils.py

```diff
@@ -67,3 +68,21 @@ def get_dbus_authenticator() -> Optional[AuthExternal]:
         auth = AuthExternal(uid=uid)
 
     return auth
+
+
+def is_running_on_gateway() -> bool:
+    """
+    Determines if the code is running on a gateway device.
+
+    Proglove Gateways have a hostname that starts with "gateway-"
+    and has 5 to 6 decimal digits after the hyphen.
+
+    Returns:
+        True if running on a gateway device, False otherwise.
+    """
+    hostname = platform.node().lower()
+    cpu_arch = platform.machine().lower()
+    return re.match(r"^gateway-\d{5,6}$", hostname) is not None and cpu_arch in (
+        "armv7l",
+        "armv6l",
```

Comment:
ah let's make it simpler :-D
```suggestion
    return re.match(r"^gateway-\d{5,6}$", hostname) is not None and cpu_arch.startswith("arm"):
```

Reply:
roger that!

Edit: done ✅